larky/src/main/java/com/verygood/security/larky/modules/utils/Reporter.java

```diff
@@ -25,6 +25,8 @@ public void reportError(StarlarkThread thread, String message) {
   }
 
   static public void report(StarlarkThread thread, String msg, Console console) {
+    // limit the size of a string to 1K (1024 characters)
+    msg = msg.length() <= 1024 ? msg : msg.substring(0, 1024);
```

Comment:
We can't do it like this after the fact. Also this needs to be configurable.

Reply:
I agree with @lapetus999, this probably needs to change from a `String` to a buffer of a configurable fixed size that gets full and starts to drop messages.